### PR TITLE
test(export): add import/export data tests

### PR DESCRIPTION
**Pull request by wwhai**
Fixes <issue-number>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

## More information

### CODE REVIEW

File: apps/emqx_web_hook/test/emqx_actions_test_utils.erl [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        _ -> error(not_cassandra_sql)
+    end;
+
+records_count([" values (" | Sql], dolphindb) ->
+    Data = [Elm || Elm <- hd(hd(Sql)), Elm =/= "[", Elm =/= "]", Elm =/= ","],
+    length(Data);
+records_count(Sql, dolphindb) when is_binary(Sql) ->
+    1;
+
+records_count("INSERT ALL" ++ Sql, oracle) ->
```
Comment (k32) [inline apps/emqx_web_hook/test/emqx_actions_test_utils.erl:144]: Nit: add trailing space after ALL to make sure the caller doesn't need to care about leading spaces. Same for the other clause

**Review by k32 — APPROVED** [review]

File: apps/emqx_web_hook/test/emqx_webhook_actions_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    Resources = emqx_rule_registry:get_resources(),
+    ?assertEqual(1, length(Resources)),
+    TestData = #resource{
+        id = <<"resource:771522">>,
+        type = web_hook,
+        config = #{<<"cacertfile">> => <<>>,<<"certfile">> => <<>>,
+                   <<"connect_timeout">> => <<"3s">>,<<"keyfile">> => <<>>,
+                   <<"method">> => <<"POST">>,<<"pool_size">> => 8,
+                   <<"request_timeout">> => <<"3s">>,
+                   <<"url">> => <<"http://www.baidu.com">>,
```
Comment (tigercl) [inline apps/emqx_web_hook/test/emqx_webhook_actions_SUITE.erl:53]: It is recommended to change the URL.

**Comment by HJianBo** [comment]
Hi @wwhai, I believe there are several keys to this test:

1. I think the xxx_test_utils.erl module is not necessary. We can just construct a complete data file for each version. E.g.: 4.0.x, 4.1.x, 4.2.x.

2. Need to verify more rigorously that the imported data is correct. (The correct definition means that the functionality appears to the user to be completely free of compatibility implications)

3. Try to cover all the functional modules that support data export.

**Review by tigercl — DISMISSED** [review]

**Comment by HJianBo** [comment]
How about we move all these test and data files under `emqx-management`? After all, it is a feature provided by emqx-management and the code we need to test is under emqx-managment

Any plans to test data import and export for 4.0.x and 4.1.x? I don't see the relevant test files in this PR?

Another suggestion, after executing this test case, we can look at the coverage of the emqx_mgmt_data_backup module, which is very important.

File: apps/emqx_bridge_mqtt/test/emqx_bridge_mqtt_SUITE_data/420.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (tigercl) [inline apps/emqx_bridge_mqtt/test/emqx_bridge_mqtt_SUITE_data/420.json:1]: Move these json file and test case to management directory

File: apps/emqx_management/test/emqx_bridge_mqtt_data_export_import_SUITE.erl [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+           remove_resource(Id);
+        <<"rpc">> ->
+           handle_config(Config, Version, rpc),
+           remove_resource(Id);
+         _ -> ok
+    end
+    end, emqx_rule_registry:get_resources()).
+
+t_import420(_) ->
+    import("/420.json", 420),
```
Comment (terry-xiaoyu) [inline apps/emqx_management/test/emqx_bridge_mqtt_data_export_import_SUITE.erl:105]:
```suggestion
    import("420.json", 420),
```

**Pull request closed without merging**